node/service/src/chain_spec/frequency_rococo.rs

```diff
@@ -0,0 +1,101 @@
+#![allow(missing_docs)]
+use cumulus_primitives_core::ParaId;
+use frequency_runtime::{AccountId, AuraId, SudoConfig, EXISTENTIAL_DEPOSIT};
+use sc_service::ChainType;
+
+/// Specialized `ChainSpec` for the normal parachain runtime.
+pub type ChainSpec = sc_service::GenericChainSpec<frequency_runtime::GenesisConfig, Extensions>;
+
+/// The default XCM version to set in genesis config.
+const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
+
+use super::{get_properties, Extensions};
+
+// pub fn load_frequency_rococo_spec() -> Result<ChainSpec, String> {
+// 	ChainSpec::from_json_bytes(&include_bytes!("../../specs/frequency_rococo.json")[..])
+// }
+
```

Comment:
This is a placeholder given we would want to control specs on testnet and directly load pre-configured ones